tests/core/asyncio/test_asyncio_subscriptions_api.py

```diff
@@ -225,11 +225,16 @@ def noop(event):
     assert len(client._connections) == 3
 
     server_c.subscribe(WaitSubscription, noop)
+    await asyncio.sleep(0.01)
     assert got_subscription.is_set() is False
     server_a.subscribe(WaitSubscription, noop)
+    await asyncio.sleep(0.01)
     assert got_subscription.is_set() is False
     server_b.subscribe(WaitSubscription, noop)
     await asyncio.sleep(0.01)
+    assert got_subscription.is_set() is False
+    client.subscribe(WaitSubscription, noop)
```

Comment:
What are your thoughts on giving `wait_until_all_remotes_subscribed_to` an extra parameter `exclude_self` to support the case where you want to use that API but **without** waiting for your own subscription?